src/main/java/org/sagebionetworks/bridge/sdk/BridgeUserClient.java

```diff
@@ -65,14 +65,10 @@ public void saveStudyParticipant(StudyParticipant participant) {
         session.checkSignedIn();
         checkNotNull(participant, "StudyParticipant cannot be null.");
 
-        post(config.getParticipantSelfApi(), participant);
-
-        if (participant.getSharingScope() != null) {
-            session.setSharingScope(participant.getSharingScope());
-        }
-        if (participant.getDataGroups() != null) {
-            session.setDataGroups(participant.getDataGroups());
-        }
+        UserSession newSession = post(config.getParticipantSelfApi(), participant, UserSession.class);
+        session.setSharingScope(newSession.getSharingScope());
+        session.setDataGroups(newSession.getDataGroups());
+        session.setConsentStatuses(newSession.getConsentStatuses());
```

Comment:
Can users update consent through the Participant API? Do we need an integration test to test this?